functests/12-test-os-imaes-namespace_rbac.sh

```diff
@@ -0,0 +1,214 @@
+      return 0
+    fi
+    sleep ${wait_secs}s
+  done
+  return 1
+}
+
+find_viewrole() {
+  # Find if the View Role is created
+
+  local wait_secs=${1:-2}
+  local max_tries=${2:-15}
+
+  for idx in $(seq 1 ${max_tries}); do
+    ((${V} >= 1)) && echo "Attempt ${idx}/${max_tries} to find view role ${VIEW_ROLE}"
+    oc get role ${VIEW_ROLE} -n ${TEST_NS}
+    if [ $? == 0 ]; then
+      ((${V} >= 1)) && echo "Found view role in Attempt ${idx}"
+      return 0
+    fi
+    sleep ${wait_secs}s
+  done
+  return 1
+}
+
+find_viewrolebinding() {
+  # Find if the View Rolebinding is created
+
+  local wait_secs=${1:-2}
+  local max_tries=${2:-15}
+
+  for idx in $(seq 1 ${max_tries}); do
+    ((${V} >= 1)) && echo "Attempt ${idx}/${max_tries} to find view rolebinding ${VIEW_ROLE}"
+    oc get rolebinding ${VIEW_ROLE} -n ${TEST_NS}
+    if [ $? == 0 ]; then
+      ((${V} >= 1)) && echo "Found view rolebinding in Attempt ${idx}"
+      return 0
+    fi
+    sleep ${wait_secs}s
+  done
+  return 1
+}
+
+quit_cleanup() {
+  if [ $1 -eq 1 ]; then
+    oc delete -n ${SSP_OP_POD_NAMESPACE} -f "${SCRIPTPATH}/common-templates-unversioned-cr.yaml"
+    oc delete clusterrole ${EDIT_ROLE}
+    oc delete ns ${TEST_NS}
+    oc delete sa ${TEST_SA}
+    exit $1
+  fi
+}
+
+trap 'quit_cleanup $?' EXIT
+
+# Create the CR and wait for it be "Ready"
+oc create -n ${SSP_OP_POD_NAMESPACE} -f "${SCRIPTPATH}/common-templates-unversioned-cr.yaml" || exit 2
+wait_for_deployment_ready ${SSP_OP_POD_NAMESPACE} 5 10 "KubevirtCommonTemplatesBundle" "Running" "Successful" || exit
+
+# Check if the namespace is created
+echo "[test_id:4494]: OS Images namespace is created"
+find_namespace || exit
+
+# Check if the edit clusterrole is created
+echo "[test_id:4584]: Edit Clusterrole is created"
+find_editclusterrole || exit
+
+# Check if view role is created
+echo "[test_id:4777]: View Role is created"
+find_viewrole || exit
+
+# Check if view rolebinding is created
+echo "[test_id:4772]: View Rolebinding is created"
+find_viewrolebinding || exit
+
+# Check if the role, rolebinding, clusterrole and namespace are recreated upon deletion
+oc delete clusterrole ${EDIT_ROLE} || exit
+oc delete ns ${TEST_NS} || exit
+
+# Check if the namespace is re-created
+echo "[test_id:4770]: OS Images namespace is re-created after deletion"
+find_namespace || exit
+
+# Check if the edit clusterrole is re-created
+echo "[test_id:4771]: Edit Clusterrole is re-created after deletion"
+find_editclusterrole || exit
+
+# Check if view role is re-created
+echo "[test_id:4773]: View Role is re-created after deletion"
+find_viewrole || exit
+
+# Check if view rolebinding is re-created
+echo "[test_id:4842]: View Rolebinding is re-created after deletion"
+find_viewrolebinding || exit
+
+# Verify ServiceAccount with view role can view PVCs
+echo "[test_id:4775]: ServiceAccounts with view role can view PVCs"
+((${V} >= 1)) && echo "Create a test service account : ${TEST_SA}"
+oc create sa ${TEST_SA} || exit
```

Comment:
I think the CI is failing because this service account needs to be created in the test namespace.